Library/Homebrew/test/livecheck/strategy/github_latest_spec.rb

```diff
@@ -0,0 +1,26 @@
+# typed: false
+# frozen_string_literal: true
+
+require "livecheck/strategy/github_latest"
+
+describe Homebrew::Livecheck::Strategy::GithubLatest do
+  subject(:github_latest) { described_class }
+
+  let(:github_download_url) { "https://github.com/example/example/releases/download/1.2.3/example-1.2.3.zip" }
+  let(:github_archive_url) { "https://github.com/example/example/archive/v1.2.3.tar.gz" }
```

Comment:
```suggestion
  let(:github_release_artifact_url) { "https://github.com/example/example/releases/download/1.2.3/example-1.2.3.zip" }
  let(:github_tag_archive_url) { "https://github.com/example/example/archive/v1.2.3.tar.gz" }
  let(:github_repository_upload_url) { "https://github.com/downloads/example/example/example-1.2.3.tar.gz" }
```

I think these names are more clear than `github_download_url` and `github_archive_url`. I've also added a repository upload URL here as well, as we should be testing that, too.